metrics-core/src/main/java/software/amazon/swage/metrics/jmx/sensor/BufferPoolSensor.java

```diff
@@ -29,7 +29,7 @@
  */
 public class BufferPoolSensor implements Sensor {
 
-    private static final long M = 1024*1024;
+    private static final double M = 1024.0 * 1024.0;
```

Comment:
I'd prefer to avoid style wars in this codebase.

If you have a reason you consider critical for this style change, or you consider it inconsistent with the rest of the style in the code base, please raise it as an issue so it can be fixed across the project. Otherwise I'd prefer we keep PR's focused on functionality.

There's a lot of introduction of whitespace across the PR, this style comment applies to most/all of them.